frontends/web/src/api/account.ts

```diff
@@ -104,7 +108,7 @@ export const getAccountsTotalBalance = (): Promise<TAccountsTotalBalanceResponse
 };
 
 export type TCoinsTotalBalance = {
-  [key: string]: IAmount;
+  [key in CoinOrTokenCode]: IAmount;
```

Comment:
Hm, I think this is not 100% correct, looking at the response it seems the the backend only returns used coins
probably should change to:

```suggestion
  [key in CoinOrTokenCode]?: IAmount;
```

I'll check the other maps in this file too...

Reply:
I'll fix this up into the first commit, TS seems happy about it and doesn't complain about any other issues when making this key optional.

Reply:
fixed